doc/guides/writing-tests.md

```diff
@@ -120,7 +120,7 @@ Avoid timers unless the test is specifically testing timers. There are multiple
 reasons for this. Mainly, they are a source of flakiness. For a thorough
 explanation go [here](https://github.com/nodejs/testing/issues/27).
 
-In the event a test needs a timer, consider using the
+In the event of a ['flaky tests' situation](https://github.com/nodejs/node/wiki/Flaky-tests), consider using the
```

Comment:
I'm not sure this make sense to me.  I can see guidance that says avoid timers but if you have to use one then consider common.platformTimeout.  But I don't think its only if the test fails intermittently.  We may have tests that just need more time on some platforms (ex arm) and if there is no other way than using a timer then even if the test fails 100% common.platformTimeout() may be the right advice